concourse/steps/update_component_deps.mako

```diff
@@ -102,6 +111,16 @@ for from_ref, to_version in determine_upgrade_prs(
     upgrade_pull_requests=upgrade_pull_requests,
     ctx_repo_base_url='${ctx_repo_base_url}',
 ):
+    configured_merge_policy = [
+        policy for policy, filter_func in merge_policies.items() if filter_func(from_ref)
+    ]
+    if len(configured_merge_policy) > 1:
+        raise RuntimeError(f'More than one merge policy applies to {from_ref.name}')
+    elif len(configured_merge_policy) == 0:
+        policy = MergePolicy('manual')
```

Comment:
since MergePolicy is an enum (I think - did not look it up), rather use MergePolicy.MANUAL